.github/workflows/turdis.yml

```diff
@@ -133,8 +133,6 @@ jobs:
           #tools/travis/install_extools.sh
           mysql -u root -h 127.0.0.1 -e 'CREATE DATABASE tg_travis;'
           mysql -u root -h 127.0.0.1 tg_travis < SQL/tgstation_schema.sql
-          mysql -u root -h 127.0.0.1 -e 'CREATE DATABASE tg_travis_prefixed;'
```

Comment:
Atomize this

Reply:
See PR description, PR is to be merged after #18859